test/message/promise_reject.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const common = require('../common');
+
+// Flags: --expose-gc
+
+Promise.reject(new Error('oops'));
+
+// Manually call GC due to possible memory contraints with attempting to
+// trigger it "naturally".
+setTimeout(common.mustCall(() => {
```

Comment:
Why is this in `common.mustCall`? Wouldn't it also be fine if it fired before the setTimeout but after all microticks?